docker-compose.yml

```diff
@@ -128,6 +137,15 @@ services:
       # - TTN_LW_GCS_BASIC_STATION_DEFAULT_LNS_URI=wss://thethings.example.com:8887
       # - TTN_LW_GCS_THE_THINGS_GATEWAY_DEFAULT_MQTT_SERVER=mqtts://thethings.example.com:8882
 
+      # If Network Server enabled, Network Server interoperability configuration:
```

Comment:
The docker-compose example always has the NS enabled, so I think it's better if we leave this out to avoid confusion.

Reply:
IDEM. consistency is more important.